### DISCUSSION TITLE
can we compare last digit of version number( skip first 3 digit)? and also can we remove the first 3 digits?

### DISCUSSION THREAD

**Discussion by KhandeManish** [wixtoolset/issues#8186] [Questions]
I'm seeking to exclude the first three digits of the version number and solely compare the fourth digit with the older version build. However, when attempting to remove the initial three digits, I encountered an error: 'The Product/@Version attribute's value, '1941...4203', is not a valid version. Legal version values should follow the format 'x.x.x.x' where each 'x' represents an integer from 0 to 65534.

**Comment by KhandeManish** [comment]
How to compare version string in the Wix ToolSet Installer build?
In the new installer, I have a version string "1942", and I want to compare it with the version string of the old installer, which is "9.0.0.1941". So, the comparison should be between "1942" and "1941". Since "1942" is greater than "1941", how can I adjust this in the WiX Toolset installer?

how can i compare both version strings in .wxs file?

**Reply by robmen** [reply]
Why do you think this is possible using the Windows Installer?

Note: Burn supports it.
